assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -63,6 +64,10 @@ export default function SettingsView() {
 		path: `/a${ accountID }w${ internalWebPropertyID }p${ profileID }/admin/view/settings`,
 	} ) );
 
+	const editViewSettingsGA4URL = useSelect( ( select ) => select( STORE_NAME ).getServiceURL( {
+		path: `/a${ accountID }p${ ga4PropertyID }/admin/streams/table/${ webDataStreamID }`,
```

Comment:
This should be encoded using `escapeURI`

`escapeURI`/a${ accountID }p${ ga4PropertyID }/admin/streams/table/${ webDataStreamID }`,`

Reply:
Also do this for the UA url

Reply:
Done